eng/targetingpacks.targets

```diff
@@ -20,11 +20,9 @@
                             '$(TargetFrameworkVersion)' == 'v$(NetCoreAppCurrentVersion)'">
```

Comment:
Remove changes from this file. I'm already taking care of them via https://github.com/dotnet/runtime/pull/88991/

Reply:
I need to keep the import of the ReadyToRun.targets file as your PR does not handle actually wiring up the in-build crossgen2 invocation, but I'll remove the rest of the changes.